AdyenDropIn/Utilities/ComponentManager+PaymentComponentBuilder.swift

```diff
@@ -438,16 +438,15 @@ private extension ComponentManager {
     }
 
     func createBLIKComponent(_ paymentMethod: BLIKPaymentMethod) -> BLIKComponent? {
-        nil
-//        let config = BLIKComponent.Configuration(
-//            style: configuration.style.formComponent,
-//            localizationParameters: configuration.localizationParameters
-//        )
-//        return BLIKComponent(
-//            paymentMethod: paymentMethod,
-//            context: context,
-//            configuration: config
-//        )
+        let config = BLIKComponentConfiguration(
+            style: configuration.style.formComponent,
+            localizationParameters: configuration.localizationParameters
+        )
+        return BLIKComponent(
```

Comment:
Is it a failing init? Otherwise you might want to make the return value of the function non-optional